control-plane/grpc/protos/pool.proto

```diff
@@ -0,0 +1,133 @@
+syntax = "proto3";
+
+import "common.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/wrappers.proto";
+
+package grpc;
```

Comment:
Do we want the package to be generically named? Will other proto file (i.e. volume.proto) also be package grpc? Alternatively we could call it pool-grpc. What do you think?

Reply:
Have split into directory versioned directories and changed it to `pool` package.